### DISCUSSION TITLE
Can't turn alignment off! A bug?

### DISCUSSION THREAD

**Discussion by jddonovan** [jomjol/AI-on-the-edge-device#2460] [Q&A] [answered]
I've defined the POI areas as well as I can:

![IMG_20230607_224756](https://github.com/jomjol/AI-on-the-edge-device/assets/11009381/b5f5eab8-2e93-4926-99ee-b7fcd78ebfde)

Yet the image gets aligned and distorted:
![IMG_20230607_224735](https://github.com/jomjol/AI-on-the-edge-device/assets/11009381/bf3e037f-02c2-4975-8d8c-c8c9453ff85a)

I'm unable to turn alignment off.. it still realigns! If I set the search radius to 0, it will hang.

![Screenshot_2023-06-07-22-52-42-282_com android chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/11009381/df420455-ec93-476d-9eca-8553ebeb0d02)

Have I understood something wrong? How to keep the photo being recognised unaltered?

**Comment by caco3** [comment]
Try to make the alignment marks smaller. And make sure you use the defaults again for the alignment.

I have it like this:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/c523fcff-8edf-4b32-9e02-cb2b4da5c90d)

**Comment by jddonovan** [comment]
Yes, I'm able to mitigate the issue with modified alignment marks, but I wish to turn off the alignment entirely as it is not needed. The setting for this has no effect. I.e. looks like a bug.

**Reply by Slider0007** [reply]
It's not a bug, but a little confusing. 
If the checkbox is not selected, the default algo is used. You have to select the checkbox and set the parameter to off to switch the algo off.

**Reply by jddonovan** [reply]
Ok, the screenshot was a bit misleading as I tried different settings.
These are the settings:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/11009381/99061564-836f-4e5d-b0d1-c48c6c3cd242)

After a fresh reboot:
![Capture](https://github.com/jomjol/AI-on-the-edge-device/assets/11009381/ab8295c7-9c68-4014-a325-d9ba84869e34)

It *still* realigns (ie. image is distorted slightly).

Hence my issue. I cannot turn off the alignment, .. the settings still show that "off" is activated. A bug?

**Reply by Slider0007** [reply]
The initial rotation which is defined in the reference image screen will be processed even you set the alignment Algo to off. If you'd like achieve that nothing happens in alignment state you have to set initial rotation to 0 as well.

**Reply by jddonovan** [reply, marked as answer]
You're right that this is probably the rotation showing in the latest screenshot -- I had already corrected the rotation to zero. This is how I had it originally, before debugging the heavy distortion issues.

The thing with the distrotion is, that it only shows only sometimes when refreshing the preview page/image. At times it has shown no distortions. I presume that it will show in the preview window either the pre-processed or post-processed, depending at what phase it is in processing the image..?

Regardless, it could be that I tripped over myself during my confusion. I did not find any obvious bugs in the code either. So consider the matter solved if you don't hear from me again. Thanks!
